### PR TITLE
Build the docs - Logging 

### LINKED ISSUE #1456: Improving the "Build the docs" logging
## Description of proposed feature
The "Build the docs" log is barely usable because it is messy.

Some things I'd like to see changed:

- Reduce the manim rendering log.
- Log the time each example rendering takes.
- Log the time rendering all examples takes.
- (Log what amount of that rendering time is required for LaTeX typesetting)  

## How can the new feature be used?
When working with examples in the docs this could be used to make decisions about how complex they should be to keep the CI time reasonable.

### PR DESCRIPTION
**Pull request by AntonBallmaier**
## Motivation
The logging while building the docs is currently hardly usable.
closes #1456

## Explanation for Changes
 ...I tried to fix that by disabling manims info logs while rendering and summarizing relevant metrics after the build is finished. 

Have a look at how the docs log now:
https://readthedocs.org/api/v2/build/13697172.txt

## Further Comments
The rendering times are saved as .csv file and can be used to easiely access rendering duration data for various purposes.

## Checklist
- [x] I have read the [Contributing Guidelines](https://docs.manim.community/en/latest/contributing.html)
- [x] I have written a descriptive PR title (see top of PR template for examples)examples in the docs
- [x] My new documentation builds, looks correctly formatted, and adds no additional build warnings

## Reviewer Checklist
- [ ] The PR title is descriptive enough
- [ ] The PR is labeled correctly
- [ ] The changelog entry is completed if necessary
- [ ] Newly added functions/classes either have a docstring or are private
- [ ] Newly added functions/classes have [tests](https://github.com/ManimCommunity/manim/wiki/Testing) added and (optional) examples in the docs
- [ ] Newly added documentation builds, looks correctly formatted, and adds no additional build warnings

### CODE REVIEW

File: docs/source/manim_directive.py [hunk 1]
```diff
@@ -231,7 +235,10 @@ def run(self):
             *user_code,
             f"{clsname}().render()",
         ]
+        start = time.time()
```
Comment (naveen521kk) [inline docs/source/manim_directive.py:238]: How about using https://docs.python.org/3/library/timeit.html than this?

**Comment by AntonBallmaier** [comment]
It seems to me that this doesn't work in the CI version, since it doesn't use the makefile :( So I probably have to refactor a lot...

File: docs/Makefile [hunk 2]
```diff
@@ -19,8 +19,9 @@ help:
 # Like make clean but also remove files generated by autosummary and
 # rendered videos.
 cleanall: clean
-	rm source/reference/*
-	rm -rf source/media
+	@rm source/reference/*
+	@rm -rf source/media
+	@rm -f rendering_times.csv
```
Comment (naveen521kk) [inline docs/Makefile:24]: can you make the change in `make.bat` also?

File: docs/source/manim_directive.py [hunk 3]
```diff
@@ -265,6 +278,57 @@ def run(self):
         return []
 
 
+rendering_times_file_path = "../rendering_times.csv"
+
+
+def _write_rendering_stats(scene_name, run_time, file_name):
+    line = ",".join(
```
Comment (naveen521kk) [inline docs/source/manim_directive.py:285]: it's a csv file right? Please use CSV https://docs.python.org/3/library/csv.html module :)

File: docs/source/manim_directive.py [hunk 4]
```diff
@@ -265,6 +278,57 @@ def run(self):
...
+        ]
+    )
+    with open(rendering_times_file_path, "a") as file:
+        file.write(line + "\n")
+
+
+def _log_rendering_times(*args):
+    try:
+        with open(rendering_times_file_path) as file:
+            data = [line.split(",") for line in file.read().splitlines()]
```
Comment (naveen521kk) [inline docs/source/manim_directive.py:299]: again, use csv module

**Review by naveen521kk — CHANGES_REQUESTED** [review]

File: docs/Makefile (hunk 2 above)
Comment (AntonBallmaier) [reply docs/Makefile:24]: There is not clearing section required in make.bat. Therefor I don't think changes are needed there.

**Comment by kolibril13** [comment]
Looks really great!

File: docs/source/manim_directive.py [hunk 5]
```diff
@@ -265,6 +279,55 @@ def run(self):
         return []
 
 
+rendering_times_file_path = "../rendering_times.csv"
```
Comment (naveen521kk) [inline docs/source/manim_directive.py:282]: Try using pathlib wherever possible, rather than os module.

File: manim/mobject/mobject.py [hunk 6]
```diff
@@ -2337,9 +2337,9 @@ def shuffle_submobjects(self, *args, **kwargs):
         Examples
         --------
 
-        .. manim:: SuffleSumobjectsExample
+        .. manim:: ShuffleSumobjectsExample
```
Comment (jsonvillanueva) [inline manim/mobject/mobject.py:2340]: You caught one spelling mistake but not the other 😛
```suggestion
        .. manim:: ShuffleSubmobjectsExample
```

File: manim/mobject/mobject.py [hunk 7]
```diff
@@ -2337,9 +2337,9 @@ def shuffle_submobjects(self, *args, **kwargs):
...
 
-            class SuffleSumobjectsExample(Scene):
+            class ShuffleSumobjectsExample(Scene):
```
Comment (jsonvillanueva) [inline manim/mobject/mobject.py:2342]:
```suggestion
            class ShuffleSubmobjectsExample(Scene):
```

File: manim/mobject/mobject.py (hunk 6 above)
Comment (AntonBallmaier) [reply manim/mobject/mobject.py:2340]: Upsi :D

**Review by naveen521kk — APPROVED** [review]
LGTM

**Pull request merged**